modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -1,6 +1,6 @@
 == Create and Configure Message Types
```

Comment:
I wonder if there s/b separate tasks for Create and Configure....

Reply:
I agree, but this is out of scope. I have created a new story for this - https://www.mulesoft.org/jira/browse/DOCS-9847